<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sales</class>
 <widget class="QWidget" name="Sales">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EconoMystique</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>../.designer/backup/img/econoLogo2.png</selectedon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #f4f4ec;
color: black;
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #365b6d;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnSales">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #365b6d;
    color: white;
    border-radius: 50px;
}
QPushButton:hover {
    background-color: #5b8ca4;
}
QPushButton:pressed {
    border-style: inset;
    background-color: #365b6d;
}
</string>
    </property>
    <property name="text">
     <string>Sales</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInventory">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #365b6d;
    color: white;
    border-radius: 50px;
}
QPushButton:hover {
    background-color: #5b8ca4;
}
QPushButton:pressed {
    border-style: inset;
    background-color: #365b6d;
}
</string>
    </property>
    <property name="text">
     <string>Inventory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPOS">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #365b6d;
    color: white;
    border-radius: 50px;
}
QPushButton:hover {
    background-color: #5b8ca4;
}
QPushButton:pressed {
    border-style: inset;
    background-color: #365b6d;
}
</string>
    </property>
    <property name="text">
     <string>POS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAccount">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #365b6d;
    color: white;
    border-radius: 50px;
}
QPushButton:hover {
    background-color: #5b8ca4;
}
QPushButton:pressed {
    border-style: inset;
    background-color: #365b6d;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/pfBtn.png</normaloff>img/pfBtn.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDashboard">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #365b6d;
    color: white;
    border-radius: 50px;
}
QPushButton:hover {
    background-color: #5b8ca4;
}
QPushButton:pressed {
    border-style: inset;
    background-color: #365b6d;
}
</string>
    </property>
    <property name="text">
     <string>EconoMystique</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/econologo_transparent_cropped.png</normaloff>img/econologo_transparent_cropped.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="SalesRecords">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>47</y>
     <width>781</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sales</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="QTabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>781</width>
     <height>491</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabToday">
    <attribute name="title">
     <string>Today</string>
    </attribute>
    <widget class="QTableWidget" name="productTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>781</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #ffffff</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblTotal">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>360</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Total</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAssessment">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>370</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background: #365b6d;
	color: white;
	border-style: outset;
	border-width: 2px;
	border-radius:10px;
	border-color: black;
}
QPushButton:hover {
    background-color: #53786d;
	border: 2px solid black;
}
QPushButton:pressed {
    border-style: inset;
    background-color: #365b6d;
}</string>
     </property>
     <property name="text">
      <string>See Assessment</string>
     </property>
    </widget>
    <widget class="QPushButton" name="forecastButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>370</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background: #365b6d;
	color: white;
	border-style: outset;
	border-width: 2px;
	border-radius:10px;
	border-color: black;
}
QPushButton:hover {
    background-color: #53786d;
	border: 2px solid black;
}
QPushButton:pressed {
    border-style: inset;
    background-color: #365b6d;
}</string>
     </property>
     <property name="text">
      <string>Generate Forecast</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/econologo_transparent_cropped.png</normaloff>img/econologo_transparent_cropped.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>15</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabMonthly">
    <attribute name="title">
     <string>Monthly</string>
    </attribute>
    <widget class="QTableWidget" name="productTable_Monthly">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>781</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #ffffff</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="cbYear_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Month:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="forecastButton_2">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>370</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background: #365b6d;
	color: white;
	border-style: outset;
	border-width: 2px;
	border-radius:10px;
	border-color: black;
}
QPushButton:hover {
    background-color: #53786d;
	border: 2px solid black;
}
QPushButton:pressed {
    border-style: inset;
    background-color: #365b6d;
}</string>
     </property>
     <property name="text">
      <string>Generate Forecast</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/econologo_transparent_cropped.png</normaloff>img/econologo_transparent_cropped.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>15</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="lblTotal_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>360</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAssessment_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>370</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background: #365b6d;
	color: white;
	border-style: outset;
	border-width: 2px;
	border-radius:10px;
	border-color: black;
}
QPushButton:hover {
    background-color: #53786d;
	border: 2px solid black;
}
QPushButton:pressed {
    border-style: inset;
    background-color: #365b6d;
}</string>
     </property>
     <property name="text">
      <string>See Assessment</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Total</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Yearly</string>
    </attribute>
    <widget class="QTableWidget" name="productTable_Yearly">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>781</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #ffffff</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="cbYear">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Year :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="forecastButton_3">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>370</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background: #365b6d;
	color: white;
	border-style: outset;
	border-width: 2px;
	border-radius:10px;
	border-color: black;
}
QPushButton:hover {
    background-color: #53786d;
	border: 2px solid black;
}
QPushButton:pressed {
    border-style: inset;
    background-color: #365b6d;
}</string>
     </property>
     <property name="text">
      <string>Generate Forecast</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/econologo_transparent_cropped.png</normaloff>img/econologo_transparent_cropped.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>15</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="lblTotal_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>360</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAssessment_3">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>370</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background: #365b6d;
	color: white;
	border-style: outset;
	border-width: 2px;
	border-radius:10px;
	border-color: black;
}
QPushButton:hover {
    background-color: #53786d;
	border: 2px solid black;
}
QPushButton:pressed {
    border-style: inset;
    background-color: #365b6d;
}</string>
     </property>
     <property name="text">
      <string>See Assessment</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Total</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
